docs/reference/modules/piece-type.md

```diff
@@ -0,0 +1,272 @@
+| [`autopublish`](#autopublish) | Boolean | Set to `true` to publish all saved edits immediately. |
+| [`label`](#label-for-doc-types) | String | The human-readable label for the doc type. |
+| [`localized`](#localized) | Boolean | Set to `false` to exclude the doc type in the locale system. |
+| [`perPage`](#perpage) | Integer | The number of pieces to include on `req.data.pieces` in each page. |
+| [`pluralLabel`](#plurallabel) | String | The plural readable label for the piece type. |
+| [`publicApiProjection`](#publicapiprojection) | Object | Piece fields to make available via a public REST API route. |
+| [`quickCreate`](#quickcreate) | Boolean | Set to `true` to add the piece type to the quick create menu. |
+| [`searchable`](#searchable) | Boolean | Set to `false` to remove the piece type from search results. |
+| `showCreate` | Boolean | Set to `false` to disable UI related to creating new pieces of that type. |
+| `showArchive` | Boolean | Set to `false` to disable UI related to archiving pieces of that type. |
+| `showDiscardDraft` | Boolean | Set to `false` to disable UI related to discarding draft pieces of that type. |
+| `showDismissSubmission` | Boolean | Set to `false` to disable UI related to dismissing draft submissions for pieces of that type. |
+| `singleton` | Boolean | Set to `true` to ensure that no one can create a new piece of that type. The global doc as only one should ever exist. |
+| [`sort`](#sort) | Object | The value for a piece type's default sort order query builder. |
+
+### `autopublish`
+
+Set `autopublish` to `true` to automatically publish any changes saved to docs of this type. There is then effectively no draft mode for this doc type.
+
+The core image and file modules use this option, for example. It eliminates the need for users to think about the distinction between draft and published content while preserving the possibility of translation for different locales.
+
+#### Example
+
+```javascript
+// modules/article-category/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  options: {
+    autopublish: true
+  },
+  // ...
+}
+```
+
+### `label`
+
+`label` should be set to a text string to be used in user interface elements related to this doc type. This includes buttons to open piece manager modals.
+
+If not set, Apostrophe will convert the module `name` meta property to a readable label by splitting the `name` on dashes and underscores, then capitalizing the first letter of each word.
+
+#### Example
+
+```javascript
+// modules/feature/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  options: {
+    label: 'Featured Article'
+  },
+  // ...
+}
+```
+
+### `localized`
+
+Defaults to `true`. If set to `false`, this doc type will _not_ be included in the locale system. This means there will be only one version of each doc, regardless of whether multiple locales (e.g., for languages or regions) are active. The "users" piece disables localization in this way.
```

Comment:
Ah, here it is! This is good, but add that there is also no distinction between draft and published, even in the database.